frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -294,11 +291,11 @@ class PopupCreateFile extends React.Component {
             onDrop={this.handleChangeFile}
             hexcolor={state.config.hexcolor}
             preview={state.uploadFilePreview}
-            multipleFiles={!state.uploadFilePreview}
+            multipleFiles
           />
 
           <FileUploadList
-            uploadFilesList={state.uploadFileList}
+            uploadFilesList={state.fileToUploadList}
```

Comment:
rename `uploadFilesList` everywhere. `fileToUploadList` is a better name even in places where there is less possible confusion between names